packages/devextreme/ts/dx.all.d.ts

```diff
@@ -9453,6 +9453,10 @@ declare module DevExpress.ui {
    * @deprecated [depNote:dxChatOptions]
    */
   export interface dxChatOptions extends WidgetOptions<dxChat> {
+    /**
+     * [descr:dxChatOptions.title]
+     */
+    title?: string;
```

Comment:
JFYI: It's better фвв changes for code and public types in separate PRs. This will reduce the number of changes and simplify the review process for API reviewers, tech writers, and developers.